lib/common.js

```diff
@@ -61,12 +61,12 @@ module.exports.baseHandler = ({ command, formatter }) => async ({
       _exec('git fetch --prune')
     }
 
-    let cmd = `git tag -il '${pattern}' --cleanup=strip --sort version:refname`
+    let cmd = `git tag -il '${pattern}' --cleanup=strip --sort=-creatordate`
```

Comment:
sort by `-committerdate` may be more suitable as most if not all our git practices center around commits